r1cs/gadgets/std/src/fields/nonnative/reduce.rs

```diff
@@ -599,7 +655,7 @@ impl<SimulationF: PrimeField, ConstraintF: PrimeField> Reducer<SimulationF, Cons
                 )?;
             } else {
                 // The length restriction for the carry
-                // Costs `bits_per_limb + surfeit + 2 - shift_per_limb` many constraints.
+                // Costs `bits_per_limb + surfeit + 2 - shift_per_limb + 1` many constraints.
```

Comment:
Was it wrong before ?

Reply:
This is the error which explains also the previous modification to the overall estimation of the number of constraints for `group_and_check_equality`

Reply:
It seems to be the linear "packing" constraint in the `limb_to_bits()`.